.github/workflows/test-ruby.yml

```diff
@@ -134,7 +128,7 @@ jobs:
       - name: Load database schema
         run: './bin/rails db:create db:schema:load db:seed'
 
-      - run: bundle exec rake rspec_chunked
+      - run: bundle exec rake
```

Comment:
Shouldn't it be `bundle exec rspec`?

Reply:
Pushed change to run rspec directly instead of via rake.